src/nearOperationFileImport.ts

```diff
@@ -0,0 +1,61 @@
+// The un-exported methods are copied from the `near-operation-file-preset` package.
+// In the future, we might choose to use the package directly, but the methods aren't directly exposed.
```

Comment:
I was hoping to use the preset source directly, but these methods are not exposed. Luckily the code doesn't contain too much logic, so I was able to just lift the methods.